src/Table/MakeExpandable.tsx

```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
 
-export interface InjectedExpandableProps {
-    onClick(): void;
+export interface InjectedExpandableProps<T extends HTMLElement> {
+    onClick(): React.MouseEventHandler<T>;
     isExpand: boolean;
 }
 
 interface State {
     isExpand: boolean;
 }
 
-export function makeExpandable<P>(Component: React.ComponentType<P & InjectedExpandableProps>) {
+export function makeExpandable<P>(Component: React.ComponentType<P & InjectedExpandableProps<HTMLTableRowElement>>) { // TODO
```

Comment:
```ts
export function makeExpandable<P, T = HTMLElement>(Component: React.ComponentType<P & InjectedExpandableProps<T>>) {
```

Reply:
Error seems didn't related with T? By the way it should be right to have generic type here.